CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing
+
+## Development
+
+Harpjs setup with
+
+* Live reload using browser-sync
+* JavaScript
+  * bundeling using webpack
+  * transpiling using babel
+* Linting using eslint
+
+
+```bash
+$ gulp
+$ open localhost:3000
+```
+
+
+## How this repo works
```

Comment:
why was this moved here? let's keep all of this in the `README.md` -- the readme of this repo viewed from github should make it very clear how to use the repo. people won't click on `CONTRIBUTING.md` even if we link to it.